Text/PresidioPIIAnonymization/notebooks/PII Anonymization.ipynb

```diff
@@ -0,0 +1,381 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from azure.core.credentials import AzureKeyCredential\n",
+    "from azure.search.documents import SearchClient\n",
+    "from IPython.display import Markdown, display\n",
+    "from requests import post, put\n",
+    "\n",
+    "def printmd(string):\n",
+    "    display(Markdown(string))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Test our text on our local running API"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "API_KEY = \"[YourSecretKeyCanBeAnything]\"\n",
```

Comment:
Consider adding more information on what this key is and how to use/obtain it. (Maybe it's my limited knowledge of PowerSkills.)

Reply:
It's copy paste from the original redame.

Reply:
There is an explanation somewhere else there... I copied it.